web-client/src/app/views/scanner/scanner.page.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { ModalController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-scanner',
+  templateUrl: './scanner.page.html',
+  styleUrls: ['./scanner.page.scss'],
+})
+export class ScannerPage implements OnInit {
+  constructor(private modalCtrl: ModalController) {}
+
+  ngOnInit() {}
+
+  scanSuccessHandler(event: any) {
+    console.log(event);
```

Comment:
I assume this is still to be implemented in a follow up PR. Is that correct?

Reply:
Yes

Reply:
I think adding a `// TODO` comment so it is clear that this is not implemented yet when reviewing and working with the code in the future will help for cases like this.